client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -69,9 +69,13 @@ export const updateMediaSuccess = ( { siteId }, mediaItem ) => ( dispatch ) => {
 	dispatchFluxUpdateMediaItemSuccess( siteId, mediaItem );
 };
 
-export const updateMediaError = ( { siteId }, error ) => (/* dispatch, getState */) => {
-	dispatchFluxUpdateMediaItemError( siteId, error );
-};
+export const updateMediaError = ( { siteId, originalMediaItem } ) => [
+	receiveMedia( siteId, originalMediaItem ),
+	removeNotice( `update-media-notice-${ originalMediaItem.ID }` ),
+	errorNotice( translate( "We weren't able to process this media item." ), {
```

Comment:
I have a nit on the wording here: I'd prefer this instead:

```suggestion
	errorNotice( translate( "We were unable to process this media item." ), {
```

but after all, I'm not a native speaker, so I wouldn't call this a blocking request. 

Maybe @saramarcondes could chime in 😉

Reply:
I'm happy to take any suggestions (that's also the reason we pinged someone from design here https://github.com/Automattic/wp-calypso/pull/46778#issuecomment-716703083).

Reply:
👍  While I left a suggestion, I'm definitely happy to defer that to design or our friends from @Automattic/editorial